README.md

```diff
@@ -56,6 +56,63 @@ There are five entrypoints to the backup module.
 4. `backup.sh`: Creates a lock directory and runs a one-off backup, then removes lock directory.
  If a lock directory exists and there is a running process, exits without doing anything.
   If a lock file exists and no running process, removes the lock directory.
-4. `schedule.py`: Creates a cron job that runs backup.py every hour and logs to `./log`. Must be run as root.
+4. `schedule.py`: Creates a cron job that runs backup.sh every hour and logs to `./log`. Must be run as root.
 5. `restore.py`:  TODO
 
+# Setup
+These are the steps followed to set up starport on the annex:
+1.  Add a bb8 user and group
+
+```
+sudo useradd bb8 -U -d /var/lib/bb8
```

Comment:
I am forgetting things, but somewhere else, in another bb8 setup script, the setup script itself set up the bb8 user.  Can we not do the same here?

Reply:
Yeah, it was designed so the program itself has to run as `bb8`, but the backup location is configurable. I hadn't actually envisioned having a bb8 user on the starport - originally I was connecting as the existing montagu user, meaning no extra setup was needed on the annex. @MartinEden added the bb8 user on the annex, which required these extra steps. 

Either we could leave the backup location configurable, and just add an extra script that installs a bb8 user that can be used if desired, or we could enforce having a bb8 user on the remote machine, i.e. actually add it in the `setup_starport` script (in which case should remove the configurable backup options from the config file.) Which do you think is preferable?